### PR TITLE
NIFI-6022 ConsumeJMS - Added try/catch to protect against secondary exception in a…

### PR DESCRIPTION
**Pull request by slyouts**
… primary exception catch clause

Thank you for submitting a contribution to Apache NiFi.

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [x] Is there a JIRA ticket associated with this PR? Is it referenced 
     in the commit message?

- [x] Does your PR title start with NIFI-XXXX where XXXX is the JIRA number you are trying to resolve? Pay particular attention to the hyphen "-" character.

- [x] Has your PR been rebased against the latest commit within the target branch (typically master)?

- [x] Is your initial contribution a single, squashed commit?

### For code changes:
- [x] Have you ensured that the full suite of tests is executed via mvn -Pcontrib-check clean install at the root nifi folder?
- [ ] Have you written or updated unit tests to verify your changes?
- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)? 
- [ ] If applicable, have you updated the LICENSE file, including the main LICENSE file under nifi-assembly?
- [ ] If applicable, have you updated the NOTICE file, including the main NOTICE file found under nifi-assembly?
- [ ] If adding new Properties, have you added .displayName in addition to .name (programmatic access) for each of the new properties?

### For documentation related changes:
- [ ] Have you ensured that format looks appropriate for the output in which it is rendered?

### Note:
Please ensure that once the PR is submitted, you check travis-ci for build issues and submit an update to your PR as soon as possible.

### CODE REVIEW

File: nifi-nar-bundles/nifi-jms-bundle/nifi-jms-processors/src/main/java/org/apache/nifi/jms/processors/JMSConsumer.java [hunk 1]
```diff
@@ -126,7 +123,11 @@ public Void doInJms(final Session session) throws JMSException {
                     // We need to call recover to ensure that in the event of
                     // abrupt end or exception the current session will stop message
                     // delivery and restart with the oldest unacknowledged message
-                    session.recover();
+                    try {
+                        session.recover();
+                    } catch (Exception e1) {
+                        // likely the session is closed...just ignore and press on with initial problem
+                    }
```
[the remark this answers is not in the document]
Comment (slyouts) [reply nifi-nar-bundles/nifi-jms-bundle/nifi-jms-processors/src/main/java/org/apache/nifi/jms/processors/JMSConsumer.java]: I was attempting to use the Apache NiFi code style standards...if I missed that mark please let me know.

File: nifi-nar-bundles/nifi-jms-bundle/nifi-jms-processors/src/main/java/org/apache/nifi/jms/processors/JMSConsumer.java [hunk 2]
```diff
@@ -126,7 +123,11 @@ public Void doInJms(final Session session) throws JMSException {
                     // We need to call recover to ensure that in the event of
                     // abrupt end or exception the current session will stop message
                     // delivery and restart with the oldest unacknowledged message
-                    session.recover();
+                    try {
+                        session.recover();
+                    } catch (Exception e1) {
+                        // likely the session is closed...just ignore and press on with initial problem
```
[the remark this answers is not in the document]
Comment (slyouts) [reply nifi-nar-bundles/nifi-jms-bundle/nifi-jms-processors/src/main/java/org/apache/nifi/jms/processors/JMSConsumer.java]: Yes.

**Comment by slyouts** [comment]
I'll be happy to rebase and squash the commits upon acceptance of this PR...unless there is some other process to handle that.

**Comment by slyouts** [comment]
Removed unnecessary format changes

**Comment by ijokarumawak** [comment]
@slyouts Thanks for the updates. LGTM +1. I'm going to merge this. Thanks again for your contribution!

**Pull request closed without merging**